cmd/entrypoint/cmd/root.go

```diff
@@ -0,0 +1,172 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	constants "github.com/cybozu-go/github-actions-controller"
+	"github.com/cybozu-go/github-actions-controller/agent"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Environments
+	podName           = os.Getenv(constants.PodNameEnvName)
+	podNamespace      = os.Getenv(constants.PodNamespaceEnvName)
+	runnerToken       = os.Getenv(constants.RunnerTokenEnvName)
+	runnerOrg         = os.Getenv(constants.RunnerOrgEnvName)
+	runnerRepo        = os.Getenv(constants.RunnerRepoEnvName)
+	extendDuration    = os.Getenv(constants.ExtendDurationEnvName)
+	slackAgentSvcName = os.Getenv(constants.SlackAgentServiceNameEnvName)
+
+	// Directory/File Paths
+	runnerDir = filepath.Join("/runner")
+	workDir   = filepath.Join(runnerDir, "_work")
+
+	extendFlagFile    = filepath.Join(os.TempDir(), "extend")
+	failureFlagFile   = filepath.Join(os.TempDir(), "failure")
+	cancelledFlagFile = filepath.Join(os.TempDir(), "cancelled")
+	successFlagFile   = filepath.Join(os.TempDir(), "success")
+
+	configCommand = filepath.Join(runnerDir, "config.sh")
+	runSVCCommand = filepath.Join(runnerDir, "bin", "runsvc.sh")
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "entrypoint",
+	Short: "GitHub Actions runner Entrypoint",
+	Long:  "GitHub Actions runner Entrypoint",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return checkEnvs()
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := os.Mkdir(workDir, 0777); err != nil {
+			return err
+		}
+
+		configArgs := []string{
+			"--unattended",
+			"--replace",
+			"--name", podName,
+			"--url", fmt.Sprintf("https://github.com/%s/%s", runnerOrg, runnerRepo),
+			"--token", runnerToken,
+			"--work", workDir,
+		}
+
+		if err := runCommand(context.TODO(), runnerDir, configCommand, configArgs...); err != nil {
+			return err
+		}
+		if err := runCommand(context.TODO(), runnerDir, runSVCCommand); err != nil {
+			return err
+		}
+
+		extend, err := annotatePods(context.TODO())
+		if err != nil {
+			return err
+		}
+		if err := slackNotify(extend); err != nil {
+			return err
+		}
```

Comment:
Please use cybozu-go/well.
```suggestion
        well.Go(func(ctx context.Context) error {
    		if err := runCommand(ctx, runnerDir, configCommand, configArgs...); err != nil {
			return err
		}
		if err := runCommand(ctx, runnerDir, runSVCCommand); err != nil {
			return err
		}

		extend, err := annotatePods(ctx)
		if err != nil {
			return err
		}
		if err := slackNotify(ctx, extend); err != nil {
			return err
		}
        return nil
    })

    well.Stop()
    return well.Wait()
```